framework/generated/generated_vulkan_feature_util.cpp

```diff
@@ -63,22 +63,50 @@ void RemoveUnsupportedFeatures(VkPhysicalDevice physicalDevice, PFN_vkGetPhysica
                 GetPhysicalDeviceFeatures2(physicalDevice, &physicalDeviceFeatures2);
                 if ((currentNext->storageBuffer16BitAccess == VK_TRUE) && (query.storageBuffer16BitAccess == VK_FALSE))
                 {
-                    GFXRECON_LOG_WARNING("Feature storageBuffer16BitAccess, which is not supported by the replay device, will not be enabled");
+                    if (remove)
+                    {
+                        GFXRECON_LOG_WARNING("Feature storageBuffer16BitAccess, which is not supported by the replay device, will not be enabled");
+                    }
+                    else
+                    {
+                        GFXRECON_LOG_WARNING("Feature storageBuffer16BitAccess requested at catpure is not supported by the replay device");
```

Comment:
"catpure" - I love this typo ;-)

I would prepare part of the string in advance and save quite a few conditionals here, would make code easier to read.

Reply:
Oops missed your comment. Will fix